test_project/settings.py

```diff
@@ -236,12 +257,12 @@
 
 CACHE_MIDDLEWARE_KEY_PREFIX = "YAK"
 
-CACHES = {
-    "default": {
-        "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
-        "LOCATION": "localhost:11211",
-    }
-}
+# CACHES = {
```

Comment:
Since this is just on the test project, I'm not sure, but do we want a TODO to fix the memcache server or just delete this code?